docs/docs-content/vertex/system-management/account-management/credentials.md

```diff
@@ -139,3 +139,119 @@ Use the following steps to delete a passkey for the admin user. We recommend you
+
+  ```bash
+  curl --insecure --location 'https://vertex.domain.example/v1/auth/syslogin' \
+   --header 'Content-Type: application/json' \
+   --data '{
+    "password": "**********",
+    "username": "**********"
+   }'
+  ```
+
+  ```json hideClipboard
+  {
+    "Authorization": "**********",
+    "IsEmailSet":true,
+    "IsEmailVerified":false,
+    "IsMfa":true,
+    "IsPasswordReset":true
+  }
+  ```
+
+However, when you enable passkeys, you cannot use the JWT token returned by the `/v1/auth/syslogin` endpoint to access the system API endpoints. To circumvent this limitation, you must use the Authorization cookie created when you authenticate into the system console through a web browser. Use the following steps to access the system API endpoints with passkeys enabled.
+
+### Prerequisites
+
+- Access to the Palette VerteX system console.  Refer to [Access the System Console](../system-management.md#access-the-system-console) for guidance on how to access the system console.
+
+- The admin user must have at least one passkey configured.
+
+- An internet browser that contains developer tools.
+
+### Steps
+
+The following steps assume you are using the Chrome browser. The steps may vary slightly if you are using a different browser.
+
+1. Log in to the system console.
+
+2. Open up the browser's developer tools. You can open the developer tools by clicking the **three-dot Menu** in the top right corner of the browser and select **More tools** and then **Developer tools**.
+
+3. Navigate to the **Application** tab in the developer tools window.
+
+4. From the **left Main Menu** select **Storage** and expand the **Local Storage** section.
+
+5. Click on the entry that matches the URL of the system console. For example, if the URL of the system console is `https://vertex.domain.example`, then click on the entry that matches `https://vertex.domain.example`.
+
+6. Copy the value of the **lscache-loginauthToken** cookie. The image below highlights the sections of importance.
+
+  ![View of the auth cookie](/vertex_account-management_credentials_browser-cache-token.png)
+
+7. Access the system console API endpoints using the Authorization header with the value of **lscache-loginauthToken** cookie. .
+
+
+### Validate
+
+You can validate that you can access the system console API endpoint by attempting a query. For example, you can query the `/v1/system/tenant` endpoint to get information about the available tenants. 
+
+
+1. Use the following `curl` command and replace the placeholder `REPLACE_ME` with the value of the **lscache-loginauthToken** cookie and use your Vertex URL instead of `vertex.domain.example` that is used in the example.
```

Comment:
```suggestion
1. Use the following `curl` command and replace the placeholder `REPLACE_ME` with the value of the **lscache-loginauthToken** cookie. Remember to use your Vertex URL instead of `vertex.domain.example`.
```